src/Sentry/NdkHandlerStrategy.cs

```diff
@@ -0,0 +1,32 @@
+namespace Sentry
+{
+    /// <summary>
+    /// Handler strategy
+    /// </summary>
+    /// 
+    public enum NdkHandlerStrategy
+    {
+        /// <summary>
+        /// default handler strategy -> value 0
+        /// </summary>
+        SENTRY_HANDLER_STRATEGY_DEFAULT,
+        /// <summary>
+        /// Handle strategy chain at start -> value 1
+        /// </summary>
+        SENTRY_HANDLER_STRATEGY_CHAIN_AT_START,
+
+
+    }
+    /// <summary>
+    /// Extension class of strategy containing relevant its actions 
+    /// </summary>
+    public static class  HandlerStrategyExtension
+    {
+#if ANDROID
```

Comment:
This means we end up with an empty class on platforms other than Android. Why would we do that?